igc_lib.py

```diff
@@ -71,7 +74,148 @@ def _rawtime_float_to_hms(timef):
 
     return hms((time/3600), (time%3600)/60, time%60)
 
-
+class Turnpoint:
+    """ single turnpoint in a task.
+    
+    Attributes: 
+        lat: latitude as a float in degrees DD.DD
+        lon: longitude as a float in degrees DD.DD
+        radius: radius of cylinder or line in km
+        sort: type of turnpoint; start_exit, start_enter, cylinder, ESS, goal_cylinder, goal_line
```

Comment:
good idea about kind, I originally tried 'type' but of course that is reserved. Didn't think of kind when looking for alternatives.